test/e2e/config/vsphere-ci.yaml

```diff
@@ -6,165 +6,106 @@
+  - name: v1.6.1
+    # Use manifest from source files
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.1/bootstrap-components.yaml"
+    type: "url"
+    contract: v1beta1
+    files:
+    - sourcePath: "../data/shared/main/v1beta1/metadata.yaml"
+    replacements:
+    - old: "imagePullPolicy: Always"
+      new: "imagePullPolicy: IfNotPresent"
 
-  - name: vsphere
-    type: InfrastructureProvider
-    versions:
-      - name: v0.7.12
-        value: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v0.7.12/infrastructure-components.yaml
-        type: url
-        contract: v1alpha3
-        replacements:
-          - old: "imagePullPolicy: Always"
-            new: "imagePullPolicy: IfNotPresent"
-        files:
-          # TODO: v1a3 cluster-template includes WORKLOAD_CONTROL_PLANE_ENDPOINT_IP
-          - sourcePath: "../data/metadata.yaml"
-          - sourcePath: "../data/infrastructure-vsphere/capi-upgrades/v1alpha3/cluster-template.yaml"
-      - name: v0.8.2
-        value: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v0.8.2/infrastructure-components.yaml
-        type: url
-        contract: v1alpha4
-        replacements:
-          - old: "imagePullPolicy: Always"
-            new: "imagePullPolicy: IfNotPresent"
-        files:
-          - sourcePath: "../data/metadata.yaml"
-          - sourcePath: "../data/infrastructure-vsphere/capi-upgrades/v1alpha4/cluster-template.yaml"
-      - name: v1.0.1
-        value: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v1.0.1/infrastructure-components.yaml
-        type: url
-        contract: v1beta1
-        replacements:
-          - old: "imagePullPolicy: Always"
-            new: "imagePullPolicy: IfNotPresent"
-        files:
-          # Add a cluster template
-          - sourcePath: "../data/infrastructure-vsphere/cluster-template.yaml"
-          - sourcePath: "../data/infrastructure-vsphere/cluster-template-remote-management.yaml"
-          - sourcePath: "../data/metadata.yaml"
+- name: kubeadm
+  type: ControlPlaneProvider
+  versions:
+  - name: v1.6.1
+    # Use manifest from source files
+    value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.1/control-plane-components.yaml"
+    type: "url"
+    contract: v1beta1
+    files:
+    - sourcePath: "../data/shared/main/v1beta1/metadata.yaml"
+    replacements:
+    - old: "imagePullPolicy: Always"
+      new: "imagePullPolicy: IfNotPresent"
+
+- name: vsphere
+  type: InfrastructureProvider
+  versions:
+  - name: v1.9.0
+    # Use manifest from source files
+    value: "https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v1.9.0/infrastructure-components.yaml"
+    type: "url"
+    contract: v1beta1
+    files:
+    # Add a cluster template
+    - sourcePath: "../../../test/e2e/data/infrastructure-vsphere/main/cluster-template.yaml"
```

Comment:
you should copy this one over / update from CAPV?

Reply:
The data folder is copied to `test/e2e/` in the Makefile so the path is valid.
```
$(E2E_DATA_DIR): $(TMP_CAPV_DIR)
	$(MAKE) -C $(TMP_CAPV_DIR) generate-e2e-templates && cp -r $(TMP_CAPV_DIR)/test/e2e/data $(E2E_DATA_DIR) && \
	cp $(TMP_CAPV_DIR)/metadata.yaml $(E2E_DATA_DIR)
```

Reply:
Ah its copying over the latest files 👍  I thought they are checked in in this repo 👍